detail.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
+
+@view_function
+def process_request(request, product:cmod.Product=None):
+
+    for p in request.last_five:
```

Comment:
A better named variable would be great here

Reply:
I agree. Thanks!